lib/slax_web/websockets/poker.ex

```diff
@@ -0,0 +1,164 @@
+defmodule SlaxWeb.Poker do
```

Comment:
is the idea that this replaces the existing poker functionality of slax?

Reply:
Yeah I initially renamed it since it is no longer a controller but split it out because I wanted to still have the controllers working just in case